sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/configuration/AadOAuth2ClientConfiguration.java

```diff
@@ -83,6 +85,19 @@ OAuth2ClientAuthenticationJwkResolver oAuth2ClientAuthenticationJwkResolver(AadA
             properties.getCredential().getClientCertificatePassword());
     }
 
+    /**
+     * Declare RestOperations bean used by various OAuth2AccessTokenResponseClient.
+     *
+     * @param builderObjectProvider the optional rest template builder bean.
+     * @return RestOperations bean
+     */
+    @Bean
+    @ConditionalOnMissingBean(RestOperations.class)
+    public RestOperations aadAuthRestOperations(ObjectProvider<RestTemplateBuilder> builderObjectProvider) {
```

Comment:
Is it better to have a global top of this bean `RestOperation`？currently， there are 2 bean definitions.

Reply:
Will do.